tsconfig.json

```diff
@@ -1,25 +1,3 @@
 {
-  "compileOnSave": false,
-  "compilerOptions": {
-    "allowUnreachableCode": false,
-    "declaration": false,
-    "downlevelIteration": true,
-    "experimentalDecorators": true,
-    "forceConsistentCasingInFileNames": true,
-    "importHelpers": true,
-    "lib": ["es2020", "dom"],
-    "module": "esnext",
-    "moduleResolution": "node",
-    "noFallthroughCasesInSwitch": true,
-    "noImplicitOverride": true,
-    "noImplicitReturns": true,
-    "noPropertyAccessFromIndexSignature": true,
-    "noUnusedLocals": true,
-    "noUnusedParameters": true,
-    "sourceMap": true,
-    "strict": true,
-    "target": "es2020",
-    "jsx": "react"
-  },
-  "exclude": ["**/node_modules/*"]
+  "extends": "./packages/tsconfig/tsconfig.json"
```

Comment:
Werkt `@nl-design-system/tsconfig` hier niet?

Reply:
Ik denk het niet want het is geen dependency. Ik denk overigens dat in geen van de configs die we aanbieden extends van andere moeten zitten. Ik was daar vanochtend nog even naar aan het kijken maar het lost volgens mij geen enkel probleem op.

Voor een React component hebben we altijd twee configuraties nodig:

1. standaard `tsconfig.json` met `"composite": true` voor het geval het component een reference _is_
2. een `tsconfig.build.json` voor wanneer het component als standalone component gebouwd wordt

Het word meteen heel erg onoverzichtelijk als de `"extends"` zelf ook nog weer extenden. De enige manier om er dan nog wijs uit te worden is expliet iets als `tsc --showConfig` af te trappen om te zien hoe de config er precies uitziet.